drivers/gnss/gnss_quectel_lcx6g.c

```diff
@@ -100,11 +89,10 @@ MODEM_CHAT_SCRIPT_NO_ABORT_DEFINE(suspend_script, suspend_script_cmds,
 				  NULL, QUECTEL_LCX6G_SCRIPT_TIMEOUT_S);
 #endif /* CONFIG_PM_DEVICE */
 
-MODEM_CHAT_MATCH_DEFINE(any_match, "", "", NULL);
 MODEM_CHAT_MATCH_DEFINE(pair062_ack_match, "$PAIR001,062,0*3F", "", NULL);
 MODEM_CHAT_SCRIPT_CMDS_DEFINE(
 	resume_script_cmds,
-	MODEM_CHAT_SCRIPT_CMD_RESP("$PAIR002*38", any_match),
+	MODEM_CHAT_SCRIPT_CMD_RESP("$PAIR002*38", modem_chat_any_match),
```

Comment:
`module` is `modem_chat` and `name` is `any_match`; what is the issue with the naming? How would name collisions happen here? (I don't think that anything else could bear the name `any_match` in `modem_chat`.)

Reply:
If we want to create a helper script chat which sends nothing and matches, then we get `modem_chat_script_chat_any_match`, or a script that sends nothing, and matches everything, we get `modem_chat_script_any_match`. I just want to keep the rules the same for the match, since `modem_chat` is the module itself.

Anyway, we can easily rename it later if necessary, let's keep `modem_chat_any_match` :)